pbtar_schema.json

```
@@ -27,6 +27,11 @@
         "description": "Detailed explanation of the scenario category",
         "type": "string"
       },
+      "net_zero_reached_year": {
+        "description": "Year by which net zero is reached in the scenario",
+        "type": ["string", "null"],
+        "pattern": "^(20(3[0-9]|[4-9][0-9])|21[0-9][0-9])$"
```

```suggestion
        "pattern": "^(20[3-9][0-9])|(21[0-9][0-9])$"
```

If the validation is to allow any value between 2030 and 2199, then I'm not sure what the advantage of the first `|` (distinguishing between the static `3` and the range `4-9`) is